lib/hal/debug.c

```diff
@@ -236,6 +236,11 @@ void debugClearScreen( void )
 	XVideoFlushFB();
 }
 
+void debugResetCursor( void ) {
+	nextRow = MARGIN;
+	nextCol = MARGIN;
```

Comment:
It would probably be better to just make this a generic "set cursor to x,y" function instead, and have resetting the cursor to 0,0 just be one of the cases handled by it.